nanoFramework.Device.Can/CanTiming.cs

```diff
@@ -0,0 +1,83 @@
+//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+namespace nanoFramework.Device.Can
+{
+    /// <summary>
+    /// Helper for common CAN timings for the STM32
+    /// </summary>
+    public static class CanTimingSTM32
+    {
+        /// <summary>
+        /// 25 KBaud.
+        /// </summary>
+        public static CanSettings Timing500KBaud () {
+            return new CanSettings(6, 8, 1, 0);
+        }
+
+    }
+
+    /// <summary>
+    /// Helper for common CAN timings for the ESP32. Values from https://github.com/espressif/esp-idf/blob/master/components/hal/include/hal/twai_types.h#L54
+    /// </summary>
+    public static class CanTimingESP32
```

Comment:
For coherence with the usual naming:

```suggestion
    public static class CanTimingEsp32
```